### PR TITLE
Wiki: New Pages & Add Icon For External Links

### PR DESCRIPTION
**Pull request by RoboticMind**
* Adds a light blue icon next to external links to differentiate them from inter-wiki and inter-site links like many wikis do
* Create a page about voting and polls
* Creates a page about the foundation

### CODE REVIEW

**Comment by RoboticMind** [comment]
@barton2526 @ShmoogleOsukami whenever you get the chance, can you take a look at this?

**Comment by RoboticMind** [comment]
@barton2526 whenever you get the chance, can you merge this?

File: wiki/foundation.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: Gridcoin Foundation
+layout: wiki
+description: Information about the Gridcoin foundation fund
+---
+
+# Introduction
+
+The Gridcoin foundation is a fund used for development, sever cost, or
```
Comment (barton2526) [inline wiki/foundation.md:9]: spelling, "server"

File: wiki/foundation.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+for any expenses or uses approved in [polls](voting "wikilink"). 
+
+The foundation is currently on the address `bc3NA8e8E3EoTL1qhRmeprbjWcmuoZ26A2`. 
+Donations there are accepted and welcome. It is a [multi-sig](https://en.wikipedia.org/wiki/Multisignature) wallet with 
+three different keyholders (iFoggz, ravonn, and quezacoatl) and requires
```
Comment (barton2526) [inline wiki/foundation.md:14]: keyholders are incorrect

File: wiki/foundation.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+to modern day Gridcoin. The coins that were not converted were originally 
+put into an address run by the pseudonymous founder Rob Halförd[^1] on the address `SFLRgjBx5HAtR5dedegRLUbQVS6ejvdu6j` (don't send coins to that address --- it is no longer in use).
+
+In 2019 it was transferred over to its current multisignature address[^2]. This
+helped deal with some of the concerns of just one person having control over 
+the funds.
+
+The longest running use of foundation funds so far has been compensating development. Originally
+developers would invoice this through CryptoCurrencyTalk[^3], but it is now done through the Gridcoin Discord
+on the `#conpensation` channel
```
Comment (barton2526) [inline wiki/foundation.md:32]: spelling, "compensation"

File: wiki/voting.md [hunk 4]
```diff
@@ -0,0 +1,551 @@
...
+The formula for exactly how it's weighted can be found [here](#11-vote-weight-and-active-vote-weight "wikilink")
+
+# Poll Requirements
+In 2019, the current standards that polls need to follow was approved. It laid out
+what types of polls there were and what each poll type needs to pass 
+
+
+All polls but the opinion/casual polls require:
+* A main discussion thread on GitHub or reddit
+* Secondary discussion threads on GitHub, Reddit, and Steemit with links to each
```
Comment (barton2526) [inline wiki/voting.md:43]: reddit is capitalized here but not above.  Should be consistent

File: wiki/voting.md [hunk 5]
```diff
@@ -0,0 +1,551 @@
...
+active vote weight (AVW) requirement. AVW is an estimate how much voting
+weight is from users who are active on the network. It is described in more detail 
+in the [active vote weight](#active-vote-weight "wikilink") section
+
+
+## Requirements for Funding
+Any poll that requests funding must state:
+* Intended use of funds
+* The date of work begin
+* The date of work end
```
Comment (barton2526) [inline wiki/voting.md:60]: awkward wording

Comment (RoboticMind) [reply wiki/voting.md]: Took that directly from the polls requirement doc, but agree that the wording there is awkward (should have looked closer at it). Changed it to 
```
Any poll that requests funding must state:
* The intended use of funds
* The date that the work begins (or began)
* The date that the work ends (or ended)
* Total amount of funding requested (in GRC)
* When funds are released
* How funds are released
* The payee's name or alias
* The payee's wallet address
* Reliable contact information
```

**Pull request merged**